src/components/FullPlayground.tsx

```diff
@@ -268,22 +267,6 @@ const useStyles = makeStyles((theme: Theme) =>
       height: "60vh",
       width: "100%",
     },
-    hideTextOnMed: {
```

Comment:
This is replaced by `className="hidden md:inline"` on the spans that wrap the text.